### PR TITLE
refactor(performance): Perf emails

### PR DESCRIPTION
**Pull request by temichus**
1 appends the backend and node type to the existing subject of Perf Emails that support
differend backends

2 move Prometheus stats to the bottom of perf emails body

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by temichus** [comment]
@roydahan @fgelcer @aleksbykov put your thoughts regarding these changes before I start testing them 

from @avikivity 
Some ergonomics:
 - move "Performance Regression Compare Results" to the end of the title. It's repetitive and doesn't contain information.
 - add the node type
 - drop the latency measurements, they're useless. At least don't start with latency on throughput tests.

**Comment by temichus** [comment]
Screenshot from 2023-06-09 17-59-53

**Review by fgelcer — DISMISSED** [review]
it will conflict with my current PR #6182 , but LGTM

**Review by aleksbykov — APPROVED** [review]
LGTM, I little concern about begin of email subject. It starts with `[perf-regression]`
May be it is better to `Perf Regression` or `Performance Regression`, WDYT?

**Comment by temichus** [comment]
> LGTM, I little concern about begin of email subject. It starts with `[perf-regression]` May be it is better to `Perf Regression` or `Performance Regression`, WDYT?

I prefer to use something without whitespaces for easy search in emails mess

**Comment by temichus** [comment]
@fruch @soyacz  Could you please merge/backport it?

File: sdcm/results_analyze/__init__.py [hunk 1]
```diff
@@ -1455,7 +1462,7 @@ def check_regression(self, test_name, test_results, test_details):  # pylint: di
             test_details=test_details,
             screenshots=test_details.pop("screenshots"),
         )
-        subject = f"Performance Regression: {test_name} - {format_timestamp(test_details['start_time'])}"
+        subject = f"[perf_regression] {test_name} - {format_timestamp(test_details['start_time'])}"
```
Comment (roydahan) [inline sdcm/results_analyze/__init__.py:1465]: Changing the subject at this point will be more painful than keeping it.
If one wants to search the inbox, one will need to search both the old subject and the new one while the short version isn't that much shorter.

I suggest not changing the subject.

File: sdcm/results_analyze/__init__.py [hunk 2]
```diff
@@ -1473,7 +1480,7 @@ def __init__(self, es_index, es_doc_type, email_recipients=(), logger=None, even
                          email_template_fp="results_search_best_throughput_config.html", logger=logger, events=events)
 
     def check_regression(self, test_name, setup_details, test_results):  # pylint: disable=too-many-locals, too-many-branches, too-many-statements
-        subject = f"Performance Regression Best throughput with configuation - {test_name} - {setup_details['start_time']}"
+        subject = f"[search_best_throughput][perf_regression] - {test_name} - {setup_details['start_time']}"
```
Comment (roydahan) [inline sdcm/results_analyze/__init__.py:1483]: same as above

**Review by roydahan — CHANGES_REQUESTED** [review]
Please do no change any subject, see my comment for it inline.

**Comment by temichus** [comment]
@roydahan,  I've updated PR now it just appends the backend and node type to the existing subject. Please check
testing in progress.

**Comment by temichus** [comment]
perf_simple_query 
Screenshot from 2023-08-17 15-25-58

**Comment by temichus** [comment]
perf tests reporting
Screenshot from 2023-08-17 16-24-59

**Review by roydahan — DISMISSED** [review]
Looks good, just update the commit message because you didnt change the title eventually.

File: microbenchmarking_test.py [hunk 3]
```diff
@@ -33,4 +33,6 @@ def test_perf_simple_query(self):
                 specific_tested_stats={"perf_simple_query_result": json.loads(output)},
                 doc_id_with_timestamp=True)
             if self.create_stats:
-                PerfSimpleQueryAnalyzer(self._test_index, self._es_doc_type).check_regression(self._test_id)
+                is_gce = bool(self.params.get('cluster_backend') == 'gce')
```
Comment (aleksbykov) [inline microbenchmarking_test.py:36]: Result of expression a == b , is bool, so no need to explicitly

**Review by aleksbykov — APPROVED** [review]

File: microbenchmarking_test.py (hunk 3 above)
Comment (temichus) [reply microbenchmarking_test.py]: fixed

**Review by roydahan — APPROVED** [review]

**Pull request merged**